aries_cloudagent/protocols/out_of_band/v1_0/manager.py

```diff
@@ -384,7 +381,7 @@ async def receive_invitation(
                             session=self._session, key="reuse_msg_state"
                         )
                 except asyncio.TimeoutError:
-                    # If no reuse_accepted or problem_report message was recieved within
```

Comment:
Above, near line 80, create_invitation() has two parameters[include_handshake, use_connections] for the handshake protocols. I propose an Enum here. I will create it, push to my repo, and message you.